### ISSUE TITLE
Compressor Update

### ISSUE THREAD

**Issue by blackearth2014** [mschack/VCV-Rack-Plugins#30]
The compressor threshold has to be set far to high near limiting to get a gain reduction meter. See attached image. As you can see I have a very low Ratio and the Threshold is cranked just to see normal readings.

![comp levels](https://user-images.githubusercontent.com/7956901/33700109-227f8480-dae5-11e7-85b2-a83335825097.png)

**Comment by mschack** [comment]
Yeah the meters are on a dB scale so it looks funny.  I may changes those to level meters
